lightsoff_api/__init__.py

```diff
@@ -171,5 +174,26 @@ def create_place_review(path: PlacePath, body: PlaceReviewBody):
     }, HTTPStatus.OK
 
 
+@app.cli.command("delete_place_review")
+@click.argument("ids", nargs=-1)
+def delete_place_review(ids):
+    if len(ids) == 0:
+        logging.warning("At least on id must be specified. Usage: flask delete_place_review 1 2 3.")
+
+    for place_review_id in ids:
```

Comment:
I think you can just retrieve all ids at once.

Then if you want to let the user know that some of them were not found, you can just log the difference between provided/retrieved ids.

Then you can delete them in a loop and commit at the end outside the loop.

This would avoid many road trip between code and database. :)

Reply:
You're right, I updated the code. And if you think it doesn't matter to log the ids not found, we can also remove this part of course.

Reply:
Keeping the log can be useful 👍